modules/Microsoft.VirtualMachineImages/imageTemplates/.test/common/deploy.test.bicep

```diff
@@ -29,23 +29,14 @@ resource resourceGroup 'Microsoft.Resources/resourceGroups@2021-04-01' = {
 
 module resourceGroupResources 'dependencies.bicep' = {
   scope: resourceGroup
-  name: '${uniqueString(deployment().name, location)}-paramNested'
+  name: '${uniqueString(resourceGroupName, location)}-paramNested'
```

Comment:
why? If I'm not mistaken we use `deployment().name` consistently across all tests.

If you have a good reason to change it - we can do this, but should be consistent across all tests

Reply:
I changed it because I have three tests to perform from the same deployment and I believe the deployment name must be unique for each.